Multisig/UI/WalletConnectV2/WalletConnectManager.swift

```diff
@@ -19,28 +19,28 @@ import UIKit
 
 class WalletConnectManager {
     static let shared = WalletConnectManager()
-
+    
     private var publishers = [AnyCancellable]()
     private var dappConnectedTrackingEvent: TrackingEvent?
-
     private let metadata = AppMetadata(
-        name: Bundle.main.displayName,
+        name: Bundle.main.displayName + " (iOS) ",
         description: "The most trusted platform to manage digital assets on Ethereum",
         url: App.configuration.services.webAppURL.absoluteString,
         icons: ["https://app.safe.global/favicons/mstile-150x150.png",
                 "https://app.safe.global/favicons/logo_120x120.png"])
-
+    
     private init() { }
-
+    
     func config() {
         Networking.configure(projectId: App.configuration.walletConnect.walletConnectProjectId,
-                             socketFactory: NativeSocketFactory())
+                             socketFactory: SocketFactory())
         Pair.configure(metadata: metadata)
+        Web3Wallet.configure(metadata: metadata, signerFactory: DefaultSignerFactory())
```

Comment:
I think this will replace our signing logic right?, otherwise we need DefaultSignerFactory to handle all key types

Reply:
I am actually not sure what this is really used for since we do not use Auth v2 at all. I had to introduce this because it is necessary for initialization of Web3Wallet.